### ISSUE TITLE
Vignette 'docs.Rmd' is non-ASCII but has no declared encoding

### ISSUE THREAD

**Issue by dmenne** [hafen/packagedocs#23]
Using the default generated docs.Rmd, I get the following error message on build_vignettes().

"Vignette 'docs.Rmd' is non-ASCII but has no declared encoding"

German Windows 10, everything is UTF by default.

packagedocs_0.4.0

**Comment by hafen** [comment]
Interesting. Could you take a look at the result of

``` r
tools::showNonASCII(readLines("docs.Rmd"))
```

and let me know what you encounter?

**Comment by dmenne** [comment]
`5: copyright: "Dieter Menne, Menne Biomed Consulting T<c3><bc>bingen"`

Bingo.  This should be "Tübingen".

**Comment by hafen** [comment]
Great! This file is yours to edit so I presume updating that solves the problem?

**Comment by dmenne** [comment]
Not really, because I do not want to switch away from UTF, and the text looks fine in my editor. Where does the copyright come from? Are non-ascii characters forbidden in the header?

[docs.zip](https://github.com/hafen/packagedocs/files/585254/docs.zip)

**Comment by hafen** [comment]
By default the header information is populated from text parsed from your DESCRIPTION file. Can you try updating the `vignette` portion of your header to this:

``` yaml
vignette: |
  %\VignetteIndexEntry{gastempt Documentation}
  %\VignetteEngine{packagedocs::redirect}
  %\VignetteEncoding{UTF-8}
```

**Comment by dmenne** [comment]
Works. Probably make it default?
There is no "Tübingen" in the DESCRIPTION file, looks like the package gets the copyright information from Github, e.g.  https://github.com/dmenne/gastempt
